src/components/views/login/login.component.ts

```diff
@@ -28,10 +28,11 @@ export class Login {
   onLogin() {
     this.loginService.login(this.username, this.password).subscribe(res => {
       var data = res.json();
-      if(data.success){
-        localStorage.setItem("auth", 'Basic ' + btoa(this.username + ':' + this.password));
+      if(data.success && data.token){
+        localStorage.setItem("auth", data.token);
```

Comment:
now that we're just setting local storage to `data.token`, we'll need to make sure on api.service.ts to comment out the use of local storage and manually do the old style basic auth until we have a legit auth endpoint.

We still might need to, in additional, store the basic auth variant of the token so we can still work with the API.